code-challenges.rb

```diff
@@ -21,7 +28,10 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
-
+def remove_all_vowels album
+    album = album.delete "aeiouAEIOU"
+end
+    p remove_all_vowels album3
```

Comment:
Logic looks great. I don't think you need to redefine your variable. You can just return the delete action.